python/sqlflow_submitter/xgboost/train.py

```diff
@@ -224,12 +232,13 @@ def save_model_to_local_file(booster, model_params, filename):
 
     model_type = type(model).__name__
     meta["type"] = model_type
-    meta = json.dumps(meta)
+    metaJson = json.dumps(meta)
 
     # Meta data is needed for saving sklearn pipeline. See here:
     # https://github.com/dmlc/xgboost/blob/d19cec70f1b40ea1e1a35101ca22e46dd4e4eecd/python-package/xgboost/sklearn.py#L356
-    booster.set_attr(scikit_learn=meta)
+    booster.set_attr(scikit_learn=metaJson)
     booster.save_model(filename)
+    save_model_metadata("model_meta.json", meta)
```

Comment:
Do not need to save the sklearn `meta`. The sklearn `meta` here is the meta data needed for sklearn model saving. It would be written into model binary when calling `booster.save_model`. We should only save the meta info which SQLFlow needs.

Reply:
done